sos/plugins/networking.py

```diff
@@ -155,6 +155,54 @@ def setup(self):
             pred=SoSPredicate(self, kmods=['ip6table_filter'])
         )
 
+        # Get devlink output
+        self.add_cmd_output([
+                    "devlink dev",
+                    "devlink dev param show",
+                    "devlink dev info",
+                    "devlink port",
+                    "devlink sb show",
+                    "devlink sb pool show",
+                    "devlink sb port pool show",
+                    "devlink sb tc bind show",
+                    "devlink -s -v trap show"
+                ])
```

Comment:
This line is over-indented (the closing ']' should align with the 's' in `self.add_cmd_output()`).

Reply:
Fixed.